frontend/src/views/Lobby.tsx

```diff
@@ -639,6 +660,54 @@ function LobbyPage() {
                 />
               </SliderContainer>
             </HoverContainerSlider>
+            <NoMarginMediumText>Room Size</NoMarginMediumText>
+            <NoMarginSubtitleText>
+              {size === 31? 'No limit' : (size === 1? '1 person' : `${size} people`)}
+            </NoMarginSubtitleText>
+            <HoverContainerSlider>
+              <HoverElementSlider
+                enabled={isHost(currentUser)}
+                onMouseEnter={() => {
+                  if (!isHost(currentUser)) {
+                    setHoverVisible(true);
+                  }
+                }}
+                onMouseLeave={() => {
+                  if (!isHost(currentUser)) {
+                    setHoverVisible(false);
+                  }
+                }}
+              />
+              <SliderContainer>
+                <Slider
+                  min={1}
+                  max={31}
+                  value={size}
+                  disabled={!isHost(currentUser)}
+                  onChange={(e) => {
+                    const { value } = e.target;
+
+                    // Set duration to undefined to allow users to clear field
+                    if (!value) {
+                      setSize(undefined);
+                    } else {
+                      const newSize = Number(value);
+                      if (newSize >= (inactiveUsers?.length || 0) + (activeUsers?.length || 0) && newSize <= 31) {
+                        if (newSize !== size) {
+                          setError('');
+                        }
+
+                        setSize(newSize);
+                      } else if (newSize < (inactiveUsers?.length || 0) + (activeUsers?.length || 0)) {
+                        setSize((inactiveUsers?.length || 0) + (activeUsers?.length || 0));
+                        setError("The room limit cannot be set below the number of connected players. Kick a player to set the limit lower.");
```

Comment:
To be honest, I might actually prefer if we didn't have an error message at all - I feel like it's pretty intuitive that the max number of players in a room is set at minimum to the number of people currently in the room. So like for example, if a group of 4 is playing and they decide they don't want to let anyone else join, they can simply drag the slider all the way to the left and it will stay at 4 as expected (without any long error message showing). 

However, if you both feel otherwise, I think that's fine to keep it as well, maybe just make it disappear immediately on change as mentioned in one of Chris's other comments

Reply:
I think in the call we discussed a couple solutions - if we decide to keep the message, I might lean towards having it disappear after the loading is finished as well. If we do that though, then maybe the message itself could be shortened? Cause otherwise it would disappear before people could finish reading it

Reply:
Yeah, I changed it so that it disappeared after loading and made the error shorter.